### ISSUE TITLE
error with multiprocessing and Pytorch 

### ISSUE THREAD

**Issue by giulioturrisi** [DLR-RM/rl-baselines3-zoo#175]
Hi, if I try to run PPO - train.py - with n_env != 1, I get the following error

with the CPU:
"Trial 0 failed because of the following error: RuntimeError('Cowardly refusing to serialize non-leaf tensor which requires_grad, since autograd does not support crossing process boundaries.  If you just want to transfer the data, call detach() on the tensor before serializing (e.g., putting it on the queue).')"

with GPU:
trial 0 failed because of the following error: RuntimeError('cuda runtime error (801) : operation not supported at /pytorch/torch/csrc/generic/StorageSharing.cpp:245')

I'm switching between the two configuration with export CUDA_VISIBLE_DEVICES="0" before launching the train.py
In order to parallelize the environments, I added in the code in the init of ExperimentManager:

vec_env_type = "subproc"

**System Info**
 * Stable Baselines3 installed by pip
 * Nvidia 2070 in WSL (windows linux subsystem), but it seems that the problem is not related to the GPU
 * Python 3.8.2
 * PyTorch version 1.4.0

**Comment by Miffyli** [comment]
Hmm this is a new type of error. I would try running other codes that use PyTorch to ensure your installation is correct. It sounds like something is wrong on that level, possibly due to the WSL2 (but doubtful). Can you also try running things without WSL2 (on Windows)?

**Comment by giulioturrisi** [comment]
Yes, on my linux machine everything works fine. I will try to understand if it is a problem with wsl2 or not. Thank you!

**Comment by giulioturrisi** [comment]
after a new download of the stable baselines/rl baselines zoo everything works fine on WSL. I did not identified the problem, but it is solved.
